go.mod

```diff
@@ -4,21 +4,21 @@ go 1.18
 
 require (
 	github.com/cosmos/cosmos-proto v1.0.0-alpha7
-	github.com/cosmos/cosmos-sdk v0.45.4
-	github.com/cosmos/ibc-go/v3 v3.0.0
+	github.com/cosmos/cosmos-sdk v0.45.6
```

Comment:
This will have no impact without bumping the replace value at the bottom to v0.45.6-ls.1

Reply:
imported module bumped this...
we can bump the replace with the next rebase